_data/cryptocurrencies.yml

```
@@ -529,7 +529,7 @@ websites:
     img: viabtc.png
     tfa:
       - totp
```

```suggestion
      - totp
      - sms
```
Based on that new doc link, ViaBTC appears to support SMS 2FA.